service-worker.js

```diff
@@ -76,6 +76,10 @@ var regexpPNG = new RegExp(/\.png$/i);
 var regexpJS = new RegExp(/\.js/i);
 var regexpCSS = new RegExp(/\.css$/i);
 var regexpSVG = new RegExp(/\.svg$/i);
+var regexpWEBM = new RegExp(/\.webm$/i);
+var regexpMP4 = new RegExp(/\.mp4$/i);
+var regexpOGG = new RegExp(/\.og[mvg]$/i);
```

Comment:
@mossroy I should probably add ogg support in the equivalent code in jQuery mode. I'll create an issue for that.